test/quantization/test_quantize_script.py

```diff
@@ -1464,6 +1464,7 @@ def forward(self, x):
                 x = torch.max(x)
                 x = torch.min(x)
                 x = torch.mean(x)
+                x = torch.sigmoid(x)
```

Comment:
looks like we have sigmoid module as well: https://pytorch.org/docs/stable/nn.html?highlight=sigmoid#torch.nn.Sigmoid
could you add a test for that?